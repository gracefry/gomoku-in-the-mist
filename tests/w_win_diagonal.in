place G10
place A1
place F9
place B2
place E8
place C3
place D7
place D4
place E6
place E5
