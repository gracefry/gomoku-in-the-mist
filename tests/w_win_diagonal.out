White wins!
G10A1F9B2E8C3D7D4E6E5
Thank you for playing!
